pkg/database/database.go

```diff
@@ -50,9 +50,7 @@ type DBParamParser interface {
 }
 
 type DBInterface interface {
-	QueryResources(ctx context.Context, kind, apiVersion, limit, continueId, continueDate string) ([]string, int64, string, error)
-	QueryNamespacedResources(ctx context.Context, kind, apiVersion, namespace, limit, continueId, continueDate string) ([]string, int64, string, error)
-	QueryNamespacedResourceByName(ctx context.Context, kind, apiVersion, namespace, name string) (string, error)
+	QueryResources(ctx context.Context, kind, apiVersion, namespace, name, limit, continueId, continueDate string) ([]string, int64, string, error)
```

Comment:
Be mindful to the line length:
```suggestion
	QueryResources(ctx context.Context, 
	    kind, apiVersion, namespace, name, limit, continueId, continueDate string) ([]string, int64, string, error)
```